.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature Request
+about: Suggest a prompt
+title: "[INSERT PROMPT HERE]"
+labels: enhancement
+assignees: stemaMSFT
+
+---
+
+- [ ] **I have checked the list of issues and believe my issue is not a duplicate.**
```

Comment:
Might be worth looking at [isssue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) as they give you better options for collecting structured input